platforms/documentation/docs/src/docs/userguide/authoring-builds/basics/plugins.adoc

```diff
@@ -407,8 +420,31 @@ include::sample[dir="snippets/plugins/multiproject/groovy", files="settings.grad
 
 You can also encapsulate the versions of external plugins by composing the build logic using your own <<sharing_build_logic_between_subprojects#sec:sharing_logic_via_convention_plugins,convention plugins>>.
 
+[[sec:root_plugins_dsl]]
+=== 3. Applying plugins from the project root
+
+You can apply plugins from the root project in a multi-project build to configure common logic and behavior across all subprojects. The root project is the project at the top of the directory hierarchy.
```

Comment:
> across all subprojects

This seems misleading. When you apply a plugin, which was declared in a parent/root project, that plugin is only applied to the current project. It does not get transitively applied to all subprojects.

```suggestion
You can apply plugins from the root project in a multi-project build to share common logic and behavior with other projects. The root project is the project at the top of the directory hierarchy.
```

Reply:
Fixed.